thread_options.h

```diff
@@ -618,6 +615,13 @@ struct thread_options_pack {
 	fio_fp64_t latency_percentile;
 	uint32_t latency_run;
 
+	/*
+	 * flow support
+	 */
+	int32_t flow_id;
+	uint32_t flow;
+	uint32_t flow_sleep;
```

Comment:
(FYI: FIO_SERVER_VER in https://github.com/axboe/fio/blob/0ce7e3e584be6f4f834fb2fec8e52deb23e1dd12/server.h#L51 would normally need upping on network format changes but it's only just been upped and we haven't cut another release so I wouldn't change it this time round).

Reply:
thanks, I wasn't aware of that

Reply:
Looks like we've had a release so we'll need this.